net.newpipe.NewPipe.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>net.newpipe.NewPipe</id>
+  <name>NewPipe (experimental Flatpak version)</name>
```

Comment:
```suggestion
  <name>NewPipe</name>
```

Reply:
have this as a note in description, this looks weird.

Reply:
done